src/libcrun/container.c

```diff
@@ -2245,24 +2258,25 @@ wait_for_process (pid_t pid, libcrun_context_t *context, int terminal_fd, int no
 
   container_exit_code = 0;
 
-  if (context->pid_file)
+  if (args->context->pid_file)
```

Comment:
Probably should throw an error if args or args->context is NULL.

Reply:
good idea.  Fixed and pushed a new version